### DISCUSSION TITLE
Change settings option

### DISCUSSION THREAD

**Discussion by aakarsh09Z** [prasanthrangan/hyprdots#1365] [Q&A] [answered]
I am trying to invert the axis of mouse scroll. Also cant change the speed of mouse cursor. Can somebody tell me how do go to the settings options.      Also if there is any documentation available for hyprdots please provide the link.

**Comment by dieBakterie** [comment, marked as answer]
In your `hyprland.conf` in the Input section `sensitivity`, `natural_scroll` and that's no Hyprdots or an overall Hyprland "issue" the Wiki is your friend Mate [read the wiki](https://wiki.hyprland.org/Configuring/Variables/#sections) There are no settings app

**Reply by aakarsh09Z** [reply]
Thanks buddy!!

**Reply by kRHYME7** [reply]
please add it to userprefs.conf so that you don't need to set it again across updates

**Reply by dieBakterie** [reply]
`./install.sh` doesn't overwrite `userprefs.conf` and `keybindings.conf` right?

**Reply by kRHYME7** [reply]
yes
